openapi/src/test/java/io/helidon/openapi/ParameterParserTest.java

```diff
@@ -0,0 +1,222 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package io.helidon.openapi;
+
+import java.util.List;
+
+import io.helidon.common.CollectionsHelper;
+import io.helidon.openapi.ParameterParser.Location;
+import io.helidon.openapi.ParameterParser.Style;
+import java.util.Collections;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * Tests for the OpenAPI parameter utility methods.
+ * <p>
+ * Note that we test the parse method that accepts a List only for the simple
+ * style. That's because a single header value can occur multiple times in the
+ * request, so getting all of them gives a List of Strings. And the header location
+ * allows only the simple style.
+ */
+public class ParameterParserTest {
+
+    public ParameterParserTest() {
+    }
+
+    @Test
+    public void testSimpleStyle() {
+        final ParameterParser parser = ParameterParser.builder("id", Location.PATH)
+                .build(); // uses default SIMPLE style
+
+        final String input = "a,b,c";
+        final List<String> expected = CollectionsHelper.listOf("a,b,c".split(","));
+
+        final List<String> result = parser.parse(input);
+        assertEquals(expected, result);
```

Comment:
Our dev guidelines mandate the use of assertThat.
In this case it would be: `assertThat(parser.parse(input), hasItems("a", "b", "c"));`